README.md

```diff
@@ -380,6 +378,24 @@ add-zsh-hook chpwd load-nvmrc
 load-nvmrc
 ```
 
+#### bash
+
+Put this into your `$HOME/.bashrc` to call `nvm use` automatically whenever you enter a directory that contains an
+`.nvmrc` file with a string telling nvm which node to use:
+
+```bash
+# place this after nvm initialization!
+cd() {
+	if [ -z "$1" ]
+	then
+		builtin cd
+	else
+		builtin cd $1
+	fi
+	[ -f ".nvmrc" ] && [ -s ".nvmrc" ] && nvm use
```

Comment:
this will cause `cd` to exit with the status of this line, instead of the previous `cd` line.

What happens if the `builtin cd` fails?